### PR TITLE
IDEM-168

### PR DESCRIPTION
**Pull request by JasminFarhi**
IMG_7700

### CODE REVIEW

**Review by bitcoinbrisbane — DISMISSED** [review]

File: app/src/components/UserDetailsHeader.tsx [hunk 1]
```diff
@@ -42,11 +42,11 @@ const UserDetailsHeader: React.FC = () => {
     const getFingerPrint = async () => {
       const key = await pgpLocalStorage.get();
       if (!key) return;
-
       const fingerPrint = trimFingerPrint(key.fingerPrint);
       setPgpTitle(fingerPrint);
     };
-    getFingerPrint();
+    const onFocus = navigation.addListener("focus", getFingerPrint);
+    return onFocus;
```
Comment (brettvniekerk) [inline app/src/components/UserDetailsHeader.tsx:49]: Could be one line

Comment (bitcoinbrisbane) [reply app/src/components/UserDetailsHeader.tsx:49]: @brettvniekerk might be helpful to @JasminFarhi to show how?

Comment (brettvniekerk) [reply app/src/components/UserDetailsHeader.tsx:49]: @JasminFarhi for return statements you don't have to predeclare a variable only to return it on the next line, you can simplify and minify your code by just returning what you would assign to that variable. In this case, you would turn lines 48 and 49 into just `return navigation.addListener("focus", getFingerPrint);`

Comment (JasminFarhi) [reply app/src/components/UserDetailsHeader.tsx]: All done, thanks for the advice! :)

**Review by brettvniekerk — APPROVED** [review]

**Comment by brettvniekerk** [comment]
Good stuff!

**Review by xiangzhe0812 — APPROVED** [review]

**Pull request merged**